dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.mpTelemetry1.1.ee-10.0.feature

```diff
@@ -1,6 +1,7 @@
 -include= ~${workspace}/cnf/resources/bnd/feature.props
 symbolicName=io.openliberty.mpTelemetry1.1.ee-10.0
 singleton=true
+IBM-API-Package: io.openliberty.microprofile.telemetry.api;type="ibm-api"
```

Comment:
The linked issue suggests that access should only be allowed for user features. If so, we want to expose this as an SPI* package so this should be `IBM-SPI-Package`.

*This is not what is generally understood by "SPI", but it is what these feature headers mean. [[reference]](https://www.ibm.com/docs/en/was-liberty/core?topic=manually-liberty-feature-manifest-files)

Reply:
It might be a good idea to open an OpenLiberty issue with your intended solution to make it easier to tell whether something like this is a mistake or intentional.

The linked internal issue lists includes the motivations and requirements, but not the intended solution.